src/zeusops_bot/cogs/zeus_upload.py

```diff
@@ -46,19 +46,27 @@ def __init__(self, bot: "ZeusopsBot", reforger_confgen: ReforgerConfigGenerator)
         input_type=discord.SlashCommandOptionType.attachment,
         required=False,
     )
+    @discord.option(
+        "strip_versions",
+        description="Automatically remove version numbers from the uploaded modlist",
+        required=False,
+    )
     async def zeus_upload(
         self,
         ctx: discord.ApplicationContext,
         scenario_id: str,
         filename: str,
         modlist: discord.Attachment | None = None,
+        strip_versions=True,
```

Comment:
```suggestion
        strip_versions: bool =True,
```

Reply:
Is the type hint required even with a literal `True` default value?